foundry/cameraPopoutAppWrapper.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Abstraction of the Application interface to be used with the Draggable class as a substitute for the app
+ * This class will represent one popout feed window and handle its positioning and draggability
+ */
+declare class CameraPopoutAppWrapper {
+
+  /**
+   * The CameraViews application that this popout belongs to
+   */
+  view: CameraViews;
+
+  /**
+   * The div element of this specific popout window
+   */
+  element: JQuery;
+
+  /**
+   * ID of the user this popout belongs to
+   */
+  userId: string;
+
+  /**
+   * @param view- The CameraViews application that this popout belongs to
+   * @param userId- ID of the user this popout belongs to
+   * @param element- The div element of this specific popout window
+   */
+  constructor(view: CameraViews, userId: string, element: JQuery);
+
+  /**
+   * Get the current position of this popout window
+   */
+  get position(): Application.Position;
+
+  /**
+   * Sets the position of this popout window
+   */
+  setPosition(position: Application.Position): void;
```

Comment:
From my understanding, most if not all of the members are optional here.

Also we should use the destructured syntax here to match foundry.js

Reply:
Updated this to be more like the destructed arguments